### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mprast**
follow-up for #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ This is the size you want to base your typography on. Typically, this will be th
 
 ### baseLineHeight
 
-This is used to determine the line height calculations in Shevy. Each line height calculated will be a multiple or dividend of this value. _It is required that this value be unitless and a number_.
+This is used to determine the line height calculations in Shevy. Each line height calculated will be a multiple or dividend of this value. Note that ShevyJS may use half-increments when necessary to keep line heights from being unreasonably large - in other words, every line height will be of the form `<some whole number> * 0.5 * baseLineHeight`. _It is required that this value be unitless and a number_.
```
Comment (kyleshevlin) [inline README.md]: How about the following?

"...multiple or dividend of `baseLineHeight / 2`. Calculating `line-height`s in increments of half the `baseLineHeight` value avoids excessively large `line-height`s for `font-size`s that barely exceed the previous increment, while still maintaining the established vertical rhythm."

I'd write it myself, but I don't want to overwrite your change and make sure you get contribution credit.

**Comment by mprast** [comment]
sgtm!

**Comment by mprast** [comment]
@kyleshevlin updated. Thanks for taking a look!

**Pull request merged**